src/libNode/Node.cpp

```diff
@@ -279,8 +312,15 @@ void Node::Init() {
   const auto& dsBlock = m_mediator.m_dsBlockChain.GetBlock(0);
   m_mediator.m_blocklinkchain.AddBlockLink(0, 0, BlockType::DS,
                                            dsBlock.GetBlockHash());
+#ifndef PRODUCTION_VERSION
```

Comment:
Is this meant to be PRODUCTION_BUILD?

Reply:
I put this in as an example to show how to read in the same accounts we use in isolated-server for testing, it means then we have a stable set of results to test against whilst playing with this rig.

If people like this facility then it should be implemented as a command line option, there is some already but with input in other formats, this uses the exact same file as the isolated server.